.github/workflows/docker_compose_works.yml

```
@@ -16,13 +16,11 @@ jobs:
           -f docker-compose.yml \
           -f .docker/docker-compose.ci.yml \
           up -d
-      with:
-        secret_gh_pages_api_token: ${{ secrets.GH_PAGES_TOKEN }}
-        git_committer_email: dev@payex.com
-        git_committer_name: payex-dev
-        gh_pages_publishing_source: gh-pages
       env:
         GITHUB_ACTOR: payex-dev
         JEKYLL_GITHUB_TOKEN : ${{ secrets.GH_PAGES_TOKEN }}
```

Can we use [`secrets.GITHUB_TOKEN`](https://help.github.com/en/actions/configuring-and-managing-workflows/authenticating-with-the-github_token#about-the-github_token-secret) instead and name the environment variable `GITHUB_TOKEN` so it's more consistent and easier to understand?

```suggestion
        GITHUB_TOKEN : ${{ secrets.GITHUB_TOKEN }}
```